ui/components/app/snaps/snap-authorship-header/snap-authorship-header.js

```diff
@@ -24,13 +24,17 @@ const SnapAuthorshipHeader = ({
   snapId,
   className,
   boxShadow = 'var(--shadow-size-lg) var(--color-shadow-default)',
+  website,
```

Comment:
This should probably be extracted to a different component instead of bloating the snap-authorship-header.

Reply:
This is now moved to a separate component, snap related component changes are reverted.